api/v1alpha3/awsmachine_webhook_test.go

```diff
@@ -119,6 +119,24 @@ func TestAWSMachine_ValidateCreate(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "additional security groups should not have filters",
+			machine: &AWSMachine{
+				Spec: AWSMachineSpec{
+					AdditionalSecurityGroups: []AWSResourceReference{
```

Comment:
I wonder that it may be better as part of v1alpha4 that we change AdditionalSecurityGroups to be a list of the security group ids and get rid of AWSResourceReference?

Reply:
Discussed in th recenet community call, see #2087

Reply:
So I assume there is no action item to be incorporated in this PR